library/globals.inc.php

```diff
@@ -2470,6 +2470,13 @@ function gblTimeZones()
             xl('Authentication SHA512 hash rounds number.')
         ),
 
+        'encryption_strategy_name' => array(
+            xl('Encryption Strategy Name') . '*',
+            'text',
+            'cryptogen',
+            xl('Name of the encryption strategy selected during installation. This setting should not be modified after installation.')
+        ),
```

Comment:
Seems odd that we even want anybody to touch this since it is set on install. is this here just to store it? If so, prob best to store it somewhere else (maybe even the version table or a new specific table that can grow if need to support new strategies (ie. if need to store things for it).

Reply:
@bradymiller hmm, I don't want anyone to touch it. I just want admins to be able to know what it's set to, without being able to see it.